apps/src/p5lab/spritelab/commands/spriteCommands.js

```diff
@@ -70,8 +65,12 @@ export const commands = {
   },
 
   makeNumSprites(num, animation) {
```

Comment:
NIT: Seeing "num" in several places starts to get a little hard to follow. It indicates that it's storing a number, but it's not quite clear what that number is used for. I'd recommend renaming it to `numSprites` or `spriteCount` or something similar.

Reply:
I agree! Will update.